src/main/native/cpp/jni/CameraServerJNI.cpp

```diff
@@ -1,17 +1,16 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "edu_wpi_cscore_CameraServerJNI.h"
```

Comment:
I thought the only issue with doing formatting on this was clang-format mangling the JNIEXPORT functions. Assuming we write a task for wpiformat to fix it, the other formatting changes done here would get applied anyway.